unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BIWLoadingAnim.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BIWLoadingAnim : MonoBehaviour
+{
+    [SerializeField]
+    private float rotationSpeed = 5f;
+
+    // Update is called once per frame
+    void Update()
+    {
+        transform.Rotate(0,0,rotationSpeed*Time.deltaTime);
+    }
```

Comment:
This is more performance than an animation, however this will disappear after Shibu implement the loading object, so this script is just a placeholder

Reply:
Are you sure? Through script we are not taking advantage of optimizations in the animation pipeline like culling and such, in the end it's a matter of how many objects are visible.

Anyway, unless we have thousands of these the performance difference can be completly ignored. I rather have an animation to avoid increasing the complexity of the project and the maintainability. In any case, as a placeholder, I dont mind.